mkosi

```diff
@@ -507,18 +511,22 @@ def mount_api_vfs(args, workspace):
             for d in paths:
                 umount(root + d)
 
+@contextlib.contextmanager
 def mount_cache(args, workspace):
-    if not args.distribution in (Distribution.fedora, Distribution.debian, Distribution.ubuntu):
-        return
-
-    if args.cache_path is None:
-        return
 
     # We can't do this in mount_image() yet, as /var itself might have to be created as a subvolume first
-    if args.distribution == Distribution.fedora:
-        mount_bind(args.cache_path, os.path.join(workspace, "root", "var/cache/dnf"))
-    elif args.distribution in (Distribution.debian, Distribution.ubuntu):
-        mount_bind(args.cache_path, os.path.join(workspace, "root", "var/cache/apt/archives"))
+    with complete_step('Mounting Package Cache'):
```

Comment:
This would announce the mounting of the package cache even if there is no such cache.